aws/resource_aws_api_gateway_integration.go

```diff
@@ -138,6 +139,13 @@ func resourceAwsApiGatewayIntegration() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+
+			"timeout_milliseconds": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				ValidateFunc: validateApiGatewayIntegrationTimeout,
```

Comment:
Minor nitpick: We have a helper function for simple integer validation: `validation.IntBetween(50, 29000)`